receiver/vcenterreceiver/metadata.yaml

```
@@ -0,0 +1,374 @@
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.cluster.vm.count:
+    enabled: true
+    description: the number of virtual machines in the cluster.
+    unit: "{virtual_machines}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [vm_count_power_state]
+  vcenter.cluster.host.count:
+    enabled: true
+    description: The number of hosts in the cluster.
+    unit: "{hosts}"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [host_effective]
+
+  vcenter.datastore.disk.usage:
+    enabled: true
+    description: The amount of space in the datastore.
+    unit: By
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: [disk_state]
+  vcenter.datastore.disk.utilization:
+    enabled: true
+    description: The utilization of the datastore.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.cpu.utilization:
+    enabled: true
+    description: The CPU utilization of the host system.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.cpu.usage:
+    enabled: true
+    description: The amount of CPU in Hz used by the host.
+    unit: "MHz"
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.disk.throughput:
+    enabled: true
+    description: The throughput to the host system's disk.
+    unit: By/s
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.disk.latency.avg:
+    enabled: true
+    description: The latency of operations to the host system's disk.
+    unit: ms
+    gauge:
+      value_type: int
+    attributes: [latency_direction, latency_type]
+    extended_documentation: This latency is the sum of the device and kernel write latencies. Requires Performance Counter level 2 for metric to populate.
+  vcenter.host.disk.latency.total:
+    enabled: true
+    description: The total reported total latency (device and kernel times)
+    unit: ms
+    gauge:
+      value_type: int
+    attributes: [latency_direction]
+  vcenter.host.memory.utilization:
+    enabled: true
+    description: The percentage of the host system's memory capacity that is being utilized.
+    unit: "%"
+    gauge:
+      value_type: double
+    attributes: []
+  vcenter.host.memory.usage:
+    enabled: true
+    description: The amount of memory the host system is using.
+    unit: MBy
+    sum:
+      monotonic: false
+      value_type: int
+      aggregation: cumulative
+    attributes: []
+  vcenter.host.network.throughput:
+    enabled: true
+    description: The amount of data that was sent or received over the network by the host.
+    unit: "{KBy/s}"
+    sum:
+      monotonic: false
```

Monotonic?

---

For this one I think its an interval based value that we are retrieving (~20s interval is the "real-time" interval for VMware) So I think we can maybe update the description to better reflect that for these? 
```suggestion
     description: The amount of data that was sent or received over the network by the host over a 20s interval.
      monotonic: false
```